code/physics/physics.cpp

```diff
@@ -371,16 +371,22 @@ void physics_sim_vel(vec3d * position, physics_info * pi, matrix *orient, vec3d*
 	if ((pi->flags & PF_SUPERCAP_WARP_IN) && (excess < SUPERCAP_WARP_EXCESS_SPD_THRESHOLD)) {
 		pi->flags &= ~(PF_SUPERCAP_WARP_IN);
 	}
-
-	// update world position from local to world coords using orient
-	vec3d world_disp;
-	vm_vec_unrotate (&world_disp, &local_disp, orient);
-	*position += world_disp;
-	*position += grav_disp;
-
-	// update world velocity
-	vm_vec_unrotate(&pi->vel, &local_v_out, orient);
-	pi->vel += grav_vel;
+	
+	if (!Scripted_velocity || !(pi->flags & PF_SCRIPTED_VELOCITY)) {
```

Comment:
Should this `||` be `&&`?  Or alternatively, remove the `Scripted_velocity` check from here and add it right before `pih->pi->flags |= PF_SCRIPTED_VELOCITY;` in the Lua API.